package.json

```diff
@@ -22,7 +22,8 @@
     "start:watch": "gulp watch",
     "start:dev": "cross-env NODE_ENV=development webpack serve --config pages/webpack.config.js",
     "start:integ": "cross-env NODE_ENV=development webpack serve --config pages/webpack.config.integ.js",
-    "prepare": "husky"
+    "prepare": "husky",
+    "postinstall": "node ./scripts/install-peer-dependency.js collection-hooks:property-filter-token-groups"
```

Comment:
This change is needed ton install the correct version of collection hooks, I will remove it before merging.

The code can be merged once https://github.com/cloudscape-design/collection-hooks/pull/74 is released.